src/pages/GroupHistory/GroupHistory.tsx

```diff
@@ -4,103 +4,29 @@ import React, {FC, useCallback, useEffect, useMemo, useState} from 'react';
-    time: string;
-    description: string;
-    category_group?: {
-        category?: ICategory
-    },
-    user: IUser
-    type: string
-}
-
-
-
-const columnHelper = createColumnHelper<GroupHistory>()
-const columns = [
-    columnHelper.accessor(`user.first_name`, {
-        header: () =>'Member',
-        cell: info => { 
-            const picture = info.row.original.user.picture ?? '' 
-            const full_name = () => {
-                if(info?.row?.original?.user?.last_name){
-                    return info.row.original.user.first_name + ' ' + info.row.original.user.last_name
-                } else {
-                    return info.row.original.user.first_name
-                }
-            }
-            const email = info.row.original.user.login
-            return info.renderValue() ?
-                <div className={classes.memberWrapper}>
-                    <div className={classes.details}>
-                        <div className={classes.icon}>
-                            <img className={classes.photo}
-                                style={{borderRadius: '50%'}}
-                                alt={'user icon'}
-                                src={isUrl(picture) ? picture : userIcon} />
-                        </div>
-                        <div className={classes.memberInfo}>
-                            <h6 className={classes.name}>{full_name()}</h6>
-                            <p className={classes.email}>{email}</p>
-                        </div>
-                    </div>
-                </div> : '-'
-            }
-    }),
-    columnHelper.accessor('category_group.category.category.title', {
-        header: () => 'Category',
-        cell: info => info.renderValue() ? <div className={classes.wrapItem}>
-            <Light
-                className={classes.dotLight}
-                style={{ display: info.row.original.type === 'expense' ? 'inline-block' : 'none' }}
-                color={info.row.original.category_group?.category?.color_code || 'var(--main-green)'}
-                type='solid' />
-            <p className={classes.itemTitle}>{info.getValue().length > 12 ? info.getValue().slice(0, 9) + '...' : info.getValue()}</p>
-        </div> : '-',
-    }),
-    columnHelper.accessor('time', {
-        header: () => 'Time',
-        cell: info => DateService.getTime(new Date(info.getValue())),
-    }),
-    columnHelper.accessor('amount', {
-        header: () => 'Amount',
-        meta: {
-            width: '100px'
-        },
-        cell: info =>
-            <p className={classes.amount} style={{ color: info.row.original.type === 'expense' ? "#FF2D55" : "#80D667", textAlign: "left" }}>{info.row.original.type === 'expense' ? "-" : "+"}${numberWithCommas(info.getValue())}</p>,
-    }),
-]
-
+import { useAppSelector } from '@hooks/storeHooks/useAppStore';
 
 
 const History: React.FC = () => {
-
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency);
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency);